coordinator/src/models/fhir/fhir-resources.ts

```diff
@@ -20,10 +22,13 @@ export interface Identifier {
   use?: string
   system?: string
   value?: string
+  extension?: Array<CodeableConceptExtension | IdentifierExtension>
```

Comment:
I take it this is to support the NHS number verification extension? I'd prefer to have a separate interface which extends Identifier. Otherwise this implies that any identifier can have an extension on it.

Reply:
Also I don't think the IdentifierExtension will be used if you make that change

Reply:
done